sormas-app/app/build.gradle

```diff
@@ -7,6 +7,11 @@ ext.sormasVersion = getVersionName()
 ext.keystoreProperties = loadKeystoreProperties()
 
 android {
+    packagingOptions{
```

Comment:
Why did this have to be removed? We should probably add a comment

Reply:
fixed